lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -143,11 +143,11 @@ def format_method?(name, node)
           return false if node.const_receiver? && !node.receiver.loc.name.is?(KERNEL)
           return false unless node.method?(name)
 
-          node.arguments.size > 1 && node.first_argument.str_type?
+          node.arguments.size > 1 && STRING_TYPES.include?(node.first_argument.type)
         end
 
         def expected_fields_count(node)
-          return :unknown unless node.str_type?
+          return :unknown unless STRING_TYPES.include?(node.type)
```

Comment:
I've notice that `STRING_TYPES.include?` appears 4 times in this class, so it can be possible to extract the private method on this occasion.

```suggestion
          return :unknown unless string_type?(node)
```

```ruby
def string_type?(node)
  STRING_TYPES.include?(node.type)
end
```